native/src/utils/PushNotificationsManager.ts

```diff
@@ -102,7 +102,7 @@ export const useForegroundPushNotificationListener = ({
         if (mounted) {
           // The CMS needs some time until the push notification is available in the API response
           setTimeout(() => {
-            // TODO IGAPP-1024: Uncomment and improve snackbar
+            // TODO IGAPP-1024 (#1979): Uncomment and improve snackbar
```

Comment:
Sorry, but it seems like this was a typo. The correct issue numbers would be IGAPP-1204 and #2140